### ISSUE TITLE
[Feature Request]: lovely design, almost perfect! some suggestions, edited

### ISSUE THREAD

**Issue by PowerfulShell** [anapnoe/stable-diffusion-webui-ux#24]
after much deep dive in this mod, you got truly UI mindset i would say, all the functions match my expectation!! really well done.
kudos to this amazing work of yours.
this is 95% of what i need, what i would have done, if i know how to code like you :) my jealousy meter is broken :)

1>!!important
bottom left:, the accordion space....
I would love to group certain area as a whole, rather than in each of their own boxes, for example, each slider have their own box, that seem unnecessary. the best example would be (hires fix), when clicked the pull down are bunch of singled out boxes, that could be better if organized, and more distinguish from the settings what it pushed down. 

2> !!important
header: quicksetting would be nicer if you can drag to the header as well, if user don't put tabs on the headers :)
maybe give the header an optional "extra row"? if applicable, 
i.e.: user need it to look like old webui header, 1st row quicksetting, 2nd row tabs
but make them smaller, not to like the original webui header, they look giant-ass huge to me, totally unnecessary.
tighter, well organize by grouping.

3>!!important
header right-side: illogical placement, 
currently the theme selector(water droplet icon) is in between the quicksetting:model selector, (theme selector), extra network, quicksetting pull out menu. the logical order should be:
---a> quicksetting:model selector
---b> 3dot quicksetting button
---c> extra network 
---d>theme selector 
another thought: (theme selector) in the far left. next to the menu button, but need to move over the theme customize menu, not worth it...i guess? maybe just set it to far right would do. :)) 

4>
in the settings menu, the left side column of menu are just bunch of texts just like original webui.... 
so an improvement would be excellent to have, 
my advice is give each one of the text menu name a left-aligned dot just before the first letter, not center to the wrapped txt.
and give it a separator underneath(excluding the last)..... that is :)

5>
lastly, would you please add a delete button to the CSS custom selector, i make two duplicates :)

**Comment by anapnoe** [comment]
3- theme configurator is optional and there is no need to be enabled once you choose your theme or make one that you like you can disable it so order is defined by importance most important right-to-left 
a) quicksettings opens all the time to finetune vae, overrides, clip, eta etc you can hit the button with closed eyes since it is the most top right on the screen
b) exra networks equally important but it can be open and closed from 2 different places so it balances with quicksettings you can open and close it from the prompt tool button or header top 
c) theme configurator is optional once you disable the extension it will disappear from the header making more available space for the tabs workflow 

5- deleting files is a destructive process I have made the function but don't want to add it cause I don't like notifications Are you sure you want to delete this file? 
the theme files are located here [extensions-builtin\sd_theme_editor\themes] to rename or delete  

4- this can be improved I agree maybe a separator can work in this case I will try it to see how it looks

2- very limited how many options fit there what if user has mobile view what monitor resolution etc quick settings can only exist on the aside view for fast access and undefined quantity, they can have custom tabs to organize them by workflow for training, extension specific, search box to find, sort by name etc you can read very easy each stack settings than reading horizontally and try to find one in a 4k monitor of course it would be nice to have the drag and drop there even for 5 more components but it is to much work maybe in the future I will implement a drag an drop system for all settings and panel to arrange them as you like e.g you can drop sampling steps on the header

1- there is an option in the user interface settings that you can arrange the order of settings but it is limited, the drag and drop system mentioned before will allow for this to be flexible and modular it will allow for grouping parameters together and accordion grouping behavior show-hide lets see what other people think

**Comment by MakingMadness** [comment]
I think it's wise to encourage one feature request or bug per issue. Grouping them in one issue makes things very disorganised and hard to mark as done. Furthermore, it's not easy for people to search the issues to see if their request has already been made by someone else. @bryanray maybe make anapnoe's life easier, separate these out and close this one?

I agree with #1, the UI could be visually grouped better and each element being in its own box is needlessly noisy. The drag and drop system sounds exciting, but for me that would be a lower priority.

**Comment by bryanray** [comment]
issue/discussion whatever it is called, this is more like a personal rant, as the title suggested it is enhancement/feature request.....
no one really cares, cause maybe is not important to others, so it is on "low priority" list of yours, but to me these things are important to my specific workflow and how my brain works, it is logical.

may better illustrated to show how it is done would make more sense, and it is all i can do, 
i dont know how to code, if i know i would do these changes myself....
i would not even ask :)

**Comment by bryanray** [comment]
yea i know those hard works...coding isnt easy, and yes lots of excitements, but in reality this mod is good as it is, really good, like 95% i am happy, these suggestion is more like push it to 100% IMO, easily said than done,
i really hope i can code do changes my sel.f

**Comment by bryanray** [comment]
i would rather do a reply like this so it not as hard to follow up than just starting over new reply....

> 3- theme configurator is optional and there is no need to be enabled once you choose your theme or make one that you like you can disable it so order is defined by importance most important right-to-left a) quicksettings opens all the time to finetune vae, overrides, clip, eta etc you can hit the button with closed eyes since it is at the most top right corner of the screen b) exra networks equally important but it can be open and closed from 2 different places so it balances with quicksettings you can open and close it from the prompt tool button or header top c) theme configurator is optional once you disable the extension it will disappear from the header making more available space for the tabs workflow
------> ok so you sorted it as your way by default
------> quicksetting constant open will block render output view cause it is on the side not the top
------>i didnt see the theme configurator in the settings, i will look for it, but i wont disable it, because it is SO GOOOD...


> 5- deleting files is a destructive process I have made the function but don't want to add it cause I don't like notifications Are you sure you want to delete this file? the theme files are located here [extensions-builtin\sd_theme_editor\themes] to rename or delete
------> maybe only to delete personal css, not the default and backup ones, also if you are worried maybe add a confirmation before action...



> 4- this can be improved I agree maybe a separator can work in this case I will try it to see how it looks
------> yes

> 2- very limited how many options fit there what if user has mobile view what monitor resolution etc quick settings can only exist on the aside view for fast access and undefined quantity, they can have custom tabs to organize them by workflow for training, extension specific, search box to find, sort by name etc you can read very easy each stack setting than reading horizontally and try to find one in a 4k monitor of course it would be nice to have the drag and drop there even for 5 more components but it is to much work maybe in the future I will implement a drag an drop system for all settings and panels to arrange them as you like e.g you can drop sampling steps on the header
------> mobile view is GOOOOD already, dont change it, mobile view isnt for serious purpose too small anyway....
------> what more impotant than drag drop top is... (((make original webui layout available as options))) this is also another reason this mod is not yet made in to the pull request and loop... it is just its own kind..... 

> 1- there is an option in the user interface settings that you can arrange the order of settings but it is limited, the drag and drop system mentioned before will allow for this to be flexible and modular it will allow for grouping parameters together and accordion grouping behavior show-hide lets see what other people think
------> i just hope for original layout as webui but can be tweak like your crazy good mod....
------>have a nice day gentlemen, love to talk to you guys...

cheers

**Comment by bryanray** [comment]
https://github.com/anapnoe/stable-diffusion-webui-ux/issues/25

The heart shape icon is back

**Comment by MakingMadness** [comment]
One of them is high priority for me, but it's knitted together with your other issues here :man_shrugging: I don't think it's important to be a coder, I just think it's worth being open to improving our feedback.

**Comment by drdancm** [comment]
How about a delete button for the last processed image, when you are experimenting and get a bunch of duds that you do not want to keep.
